QATCH/QModel/models/qmodel_v7/spacing_prior.py

```
@@ -0,0 +1,273 @@
+    def fit(
+        configs_sec: np.ndarray,
+        frac_blend: float = 0.5,
+        bound_lo_pct: float = 0.5,
+        bound_hi_pct: float = 99.5,
+    ) -> "SpacingPrior":
+        """Fit from complete configurations.
+
+        configs_sec: (N, P) array of POI times in seconds, strictly ascending
+        rows (complete fills only). P must equal len(POI_ORDER).
+        """
+        N, P = configs_sec.shape
```

Ruff (0.15.20)

[warning] Unpacked variable `N` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)